README.md

```diff
@@ -180,7 +180,7 @@ The following shows the techniqual details:
 |---|---|---|---|
 |wasmtime|:heavy_check_mark:|:heavy_check_mark:||
 |wamr(wasm-micro-runtime)|:heavy_check_mark:|:heavy_check_mark:||
-|wazero|:construction: (stdin unsupported)|:heavy_check_mark:|non-blocking stdin doesn't seem to work|
+|wazero|:heavy_check_mark:|:heavy_check_mark:||
```

Comment:
thanks to @ktock @evacchi and @achille-roussel for collaboration on this check mark!

Reply:
Thank you for your great work!